apps/complete_registration/src/forms/lunch_tickets/lunch_ticket_create.page.tsx

```diff
@@ -0,0 +1,243 @@
+import { useState, useEffect } from "react"
+import { FormWrapper } from "../FormWrapper"
+import { InputText } from "primereact/inputtext"
+import { Dropdown } from "primereact/dropdown"
+import { Checkbox } from "primereact/checkbox"
+import { Button } from "primereact/button"
+import { Badge  } from "primereact/badge"
+import {LunchTicket, validateLunchTicket} from "../../utils/lunch_tickets/lunch_tickets.utils"
+import { buildURLEncodedPayload } from "../../utils/django_format/django_format.utils"
+import { useSelector } from "react-redux"
+import { selectCompanyName } from "../../store/company/company_selectors"
+import "./lunch_ticket.css"
+import { HOST } from "../../shared/vars"
+import { RootState } from "../../store/store"
+import { selectField } from "../../store/form/form_selectors"
+
+export function CreateLunchTicketsPage() {
+    const result_unassigned_lunch_tickets = useSelector((state: RootState) =>
+        selectField(state, "unassigned_lunch_tickets")
+    );
+    const result_fair_days = useSelector((state: RootState) =>
+        selectField(state, "fair_days")
+    );
+    const result_lunch_times = useSelector((state: RootState) =>
+        selectField(state, "lunch_times")
+    );
+    const result_dietary_restrictions = useSelector((state: RootState) =>
+        selectField(state, "dietary_restrictions")
+    );
+
+    const unassigned_lunch_tickets = (result_unassigned_lunch_tickets?.value ?? []) as number
+    const fair_days = (result_fair_days?.value ?? [""]) as string[]
+    const lunch_times = (result_lunch_times?.value ?? [""]) as string[]
+    const dietary_restrictions = (result_dietary_restrictions?.value ?? []) as string[]
+
+    const initialDietaryRestrictions: { [key: string]: boolean } = {};
+
+    dietary_restrictions.forEach(name => {
+        initialDietaryRestrictions[name] = false;
+    });
+
+
+    const [DateState, setDateState] = useState<string>(fair_days[0])
+    const [TimeState, setTimeState] = useState<string>(lunch_times[0])
+    const [Email, setEmail] = useState<string>("")
+    const [DietaryRestrictions, setDietaryRestrictions] =
+        useState<{ [key: string]: boolean }>(initialDietaryRestrictions)
+    const [OtherDietRestrictions, setOtherDietRestrictions] =
+        useState<string>("")
+    const [OtherComments, setOtherComments] = useState<string>("")
+    const [ErrorString, setErrorString] = useState<string>("")
+
+    useEffect(() => {}, [DietaryRestrictions])
```

Comment:
The useState or the useEffect? I see you deleted the use effect already. The useState, I believe they are since we need that information to be on the form. This is my way to collect them without accessing the DOM. I might be wrong and there could be better approaches.